### PR TITLE
Ornithids Post-Launch Part 1: "This Should've Been in the Main Launch"

### PR DESCRIPTION
**Pull request by Wollywoger**
## content added
this fixes and enables tailcode for ornithids, and adds four bird masks
two bird tails: 
eagle, an average length tail
swallow, a shorter tail more akin to common songbirds

four masks:
cardinal: a red, straight beak mask
secretary: a white, slightly hooked beak mask
toucan: a colorful, large beak mask that takes from its name sake
and the blue jay: a full-face covering blue mask with gas mask filters and a visor built in.
all except the bluejay are statistically identical to the stock breathmask, the bluejay is a slightly modified plague doctor mask, which is already obtainable from the loadout & costume vendors

speaking of loadout, the 4 masks can currently only be obtained from the loadout.

## non-module changes
this re-enables the previously unused tail code added in the main ornithid PR. like before, they'll be marked with "NON MODULE CHANGE"

### CODE REVIEW

**Comment by Wollywoger** [comment]
image


showing off the masks and tails.
first and last on the top is eagle, middle is swallow tail.
from left to right:
cardinal, secretary, toucan, bluejay

File: maplestation_modules/code/modules/clothing/masks/ornithid_masks.dm [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+*/
+
+/obj/item/clothing/mask/breath/ornithid
+	name = "ornithid mask"
+	desc = "please report any fowl play, such as this exact mask appearing in game, to the git!" // pun intentional
+	icon = 'maplestation_modules/icons/obj/clothing/masks.dmi'
+	icon_state = "cardinal" // placeholder so missing icons shuts up
+	worn_icon = 'maplestation_modules/icons/mob/clothing/masks.dmi'
+	/* lefthand_file = 'maplestation_modules/icons/mob/inhands/clothing/masks_lefthand.dmi'
+	righthand_file = 'maplestation_modules/icons/mob/inhands/clothing/masks_righthand.dmi'  */ // not really gonna bother making these unless i'm in a particularly creative mood Yellow TODO: actually make these
```
Comment (MrMelbert) [inline maplestation_modules/code/modules/clothing/masks/ornithid_masks.dm:12]:
```suggestion
	inhand_icon_state = null
```
I think this is just fine?

File: code/__HELPERS/global_lists.dm [hunk 2]
```diff
@@ -34,7 +34,7 @@
 	init_sprite_accessory_subtypes(/datum/sprite_accessory/pod_hair, GLOB.pod_hair_list)
 	init_sprite_accessory_subtypes(/datum/sprite_accessory/head_tentacles, GLOB.head_tentacles_list) //NON-MODULE CHANGE
 	init_sprite_accessory_subtypes(/datum/sprite_accessory/arm_wings, GLOB.arm_wings_list) //NON-MODULE CHANGE
-	// init_sprite_accessory_subtypes(/datum/sprite_accessory/tails/avian, GLOB.tails_list_avian) //NON-MODULE CHANGE (unused)
+	init_sprite_accessory_subtypes(/datum/sprite_accessory/tails/avian, GLOB.tails_list_avian) //NON-MODULE CHANGE (unused)
```
Comment (MrMelbert) [inline code/__HELPERS/global_lists.dm:37]: This comment needs an update?

**Review by MrMelbert — APPROVED** [review]
Looks fine, small things

File: code/__HELPERS/global_lists.dm (hunk 2 above)
Comment (Wollywoger) [reply code/__HELPERS/global_lists.dm]: done

File: code/__DEFINES/DNA.dm [hunk 3]
```diff
@@ -59,8 +59,8 @@
 #define DNA_POD_HAIR_BLOCK 15
 #define DNA_HEAD_TENTACLES_BLOCK 16 // NON-MODULE CHANGE
 #define DNA_ARM_WINGS_BLOCK 17 // NON-MODULE CHANGE
-// #define DNA_AVIAN_TAIL_BLOCK 18 // NON-MODULE CHANGE this, alongside other ornithid tailcode is going temporarily unused because tail code sucks
 #define DNA_AVIAN_EARS_BLOCK 18 // NON-MODULE CHANGE
+#define DNA_AVIAN_TAIL_BLOCK 19 // NON-MODULE CHANGE this, alongside other ornithid tailcode is going temporarily unused because tail code sucks
```
[the remark this answers is not in the document]
Comment (Wollywoger) [reply code/__DEFINES/DNA.dm]: done

File: code/__DEFINES/DNA.dm [hunk 4]
```diff
@@ -59,9 +59,8 @@
 #define DNA_POD_HAIR_BLOCK 15
 #define DNA_HEAD_TENTACLES_BLOCK 16 // NON-MODULE CHANGE
 #define DNA_ARM_WINGS_BLOCK 17 // NON-MODULE CHANGE
-// #define DNA_AVIAN_TAIL_BLOCK 18 // NON-MODULE CHANGE this, alongside other ornithid tailcode is going temporarily unused because tail code sucks
 #define DNA_AVIAN_EARS_BLOCK 18 // NON-MODULE CHANGE
-
+#define DNA_AVIAN_TAIL_BLOCK 19 // NON-MODULE CHANGE
 #define DNA_SEQUENCE_LENGTH 4
```
Comment (MrMelbert) [inline code/__DEFINES/DNA.dm:64]:
```suggestion
#define DNA_AVIAN_TAIL_BLOCK 19 // NON-MODULE CHANGE

#define DNA_SEQUENCE_LENGTH 4
```

**Pull request merged**